README.md

```diff
@@ -64,6 +64,7 @@ For more details, go [here](https://docs.multiversx.com/sdk-and-tools/proxy/).
 ### validator
 
 - `/v1.0/validator/statistics`     (GET) --> returns the validator statistics data from an observer from any shard. Has a cache to avoid many requests
+- `/v1.0/validator/auction`        (GET) --> returns the validator auction list data from an observer from metachain. Has a cache to avoid many requests
```

Comment:
you can replace `Has a cache to avoid many requests` with `It doesn't have a cache mechanism, since there is already one in place at the node level`. 
However, this is debatable. The data is cached at the node level, that's true. However, a cache at the proxy level would help by avoiding an additional API call to an observer

Reply:
Right, good suggestion. Refactored as suggested